<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCPropertiesEditor</class>
 <widget class="QDialog" name="VCPropertiesEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Virtual Console properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="m_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Layout">
      <attribute name="title">
       <string>Layout</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="m_keyboardGroup">
         <property name="title">
          <string>Keyboard in operate mode</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QCheckBox" name="m_grabKeyboardCheck">
            <property name="toolTip">
             <string>Make virtual console to take exclusive keyboard control during operate mode</string>
            </property>
            <property name="text">
             <string>Grab keyboard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_keyRepeatOffCheck">
            <property name="toolTip">
             <string>Prevent flickering when a keyboard hotkey has been pressed in operate mode</string>
            </property>
            <property name="text">
             <string>Turn off key repeat</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="m_gridGroup">
         <property name="toolTip">
          <string>Use a grid layout for virtual console widgets</string>
         </property>
         <property name="title">
          <string>Grid</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Horizontal (X)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_gridXSpin">
            <property name="toolTip">
             <string>Widget grid layout X resolution</string>
            </property>
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Vertical (Y)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="m_gridYSpin">
            <property name="toolTip">
             <string>Widget grid layout Y resolution</string>
            </property>
            <property name="suffix">
             <string>px</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>111</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>111</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="GrandMaster">
      <attribute name="title">
       <string>Grand Master</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="m_gmChannelsGroup">
         <property name="title">
          <string>Channels</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="m_gmIntensityRadio">
            <property name="text">
             <string>Intensity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_gmAllChannelsRadio">
            <property name="text">
             <string>All channels</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="m_gmValuesGroup">
         <property name="title">
          <string>Values</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="m_gmReduceRadio">
            <property name="text">
             <string>Reduce</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_gmLimitRadio">
            <property name="text">
             <string>Limit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>External Input</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="m_gmInputUniverseLabel">
            <property name="text">
             <string>Input Universe</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLineEdit" name="m_gmInputUniverseEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_gmInputChannelLabel">
            <property name="text">
             <string>Input Channel</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QLineEdit" name="m_gmInputChannelEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>165</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="m_autoDetectGrandMasterInputButton">
            <property name="text">
             <string>Auto Detect</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="m_chooseGrandMasterInputButton">
            <property name="text">
             <string>Choose...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DefaultSliders">
      <attribute name="title">
       <string>Default Fade</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="m_defaultFadeGroup">
         <property name="title">
          <string>Default fade slider</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="m_lowFadeLabel">
            <property name="text">
             <string>Low limit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QSpinBox" name="m_fadeLowSpin">
            <property name="toolTip">
             <string>Lowest value that can be set to the 'fade' bus with the default sliders</string>
            </property>
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="maximum">
             <number>998</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_highFadeLabel">
            <property name="text">
             <string>High limit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QSpinBox" name="m_fadeHighSpin">
            <property name="toolTip">
             <string>Highest value that can be set to the 'fade' bus with the default sliders</string>
            </property>
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Input universe</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLineEdit" name="m_fadeInputUniverseEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Input channel</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QLineEdit" name="m_fadeInputChannelEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>117</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="m_autoDetectFadeInputButton">
            <property name="toolTip">
             <string>When toggled, you can move an external slider/knob to assign it to the default fade slider.</string>
            </property>
            <property name="text">
             <string>Auto Detect</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="m_chooseFadeInputButton">
            <property name="toolTip">
             <string>Choose an external input universe &amp; channel that the default fade slider should listen to.</string>
            </property>
            <property name="text">
             <string>Choose...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Default Hold</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="m_defaultHoldGroup">
         <property name="title">
          <string>Default hold slider</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="m_lowHoldLabel">
            <property name="text">
             <string>Low limit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QSpinBox" name="m_holdLowSpin">
            <property name="toolTip">
             <string>Lowest value that can be set to the 'hold' bus with the default sliders</string>
            </property>
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="maximum">
             <number>998</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_highHoldLabel">
            <property name="text">
             <string>High limit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QSpinBox" name="m_holdHighSpin">
            <property name="toolTip">
             <string>Highest value that can be set to the 'hold' bus with the default sliders</string>
            </property>
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Input universe</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLineEdit" name="m_holdInputUniverseEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Input channel</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QLineEdit" name="m_holdInputChannelEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>117</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="m_autoDetectHoldInputButton">
            <property name="toolTip">
             <string>When toggled, you can move an external slider/knob to assign it to the default hold slider.</string>
            </property>
            <property name="text">
             <string>Auto Detect</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="m_chooseHoldInputButton">
            <property name="toolTip">
             <string>Choose an external input universe &amp; channel that the default hold slider should listen to.</string>
            </property>
            <property name="text">
             <string>Choose...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_grabKeyboardCheck</sender>
   <signal>clicked()</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotGrabKeyboardClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_keyRepeatOffCheck</sender>
   <signal>clicked()</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotKeyRepeatOffClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gridGroup</sender>
   <signal>clicked()</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotGridClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gridXSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotGridXChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gridYSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotGridYChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fadeLowSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotFadeLimitsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fadeHighSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotFadeLimitsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_autoDetectGrandMasterInputButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotAutoDetectGrandMasterInputToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_chooseGrandMasterInputButton</sender>
   <signal>clicked()</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotChooseGrandMasterInputClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_autoDetectFadeInputButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotAutoDetectFadeInputToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_chooseFadeInputButton</sender>
   <signal>clicked()</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotChooseFadeInputClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_autoDetectHoldInputButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotAutoDetectHoldInputToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_chooseHoldInputButton</sender>
   <signal>clicked()</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotChooseHoldInputClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gmIntensityRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotGrandMasterIntensityToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_gmReduceRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>VCPropertiesEditor</receiver>
   <slot>slotGrandMasterReduceToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotGrabKeyboardClicked()</slot>
  <slot>slotKeyRepeatOffClicked()</slot>
  <slot>slotGridClicked()</slot>
  <slot>slotGridXChanged(int)</slot>
  <slot>slotGridYChanged(int)</slot>
  <slot>slotFadeLimitsChanged()</slot>
  <slot>slotHoldLimitsChanged()</slot>
  <slot>slotAutoDetectGrandMasterInputToggled(bool)</slot>
  <slot>slotChooseGrandMasterInputClicked()</slot>
  <slot>slotAutoDetectFadeInputToggled(bool)</slot>
  <slot>slotChooseFadeInputClicked()</slot>
  <slot>slotAutoDetectHoldInputToggled(bool)</slot>
  <slot>slotChooseHoldInputClicked()</slot>
  <slot>slotGrandMasterIntensityToggled(bool)</slot>
  <slot>slotGrandMasterReduceToggled(bool)</slot>
 </slots>
</ui>
